dissect/target/plugins/apps/ssh/sshd.py

```diff
@@ -0,0 +1,179 @@
+from typing import TYPE_CHECKING, Any, Callable, Iterator, Optional, Union
```

Comment:
Shouldn't this file be called `opensshd.py` as not to collide with other types of sshd servers?